docs/website/docs/plus/production/prefect-integration.md

```diff
@@ -0,0 +1,246 @@
+
+## Key features
+
+- **Prefect Collector:** a dedicated way to do real-time [progress monitoring] and summary reports after each pipeline stage
+- **Retry integration:** retries without loosing intermediate results, execute custom code between retries
+- **Source decomposition flows:** Create tasks and flows that will execute your pipelines in parallel with the right settings
+
+## Prefect Collector
+
+The `PrefectCollector` creates rich artifacts in the Prefect UI to monitor dlt pipeline progress and results. Simply pass it to your pipeline's `progress` parameter to enable detailed monitoring.
+
+```py
+from dlt_plus._runner.prefect_collector import PrefectCollector
+
+@task
+def my_dlt_task():
+    pipeline = dlt.pipeline(
+        pipeline_name="my_pipeline",
+        destination="duckdb",
+        dataset_name="my_dataset",
+        progress=PrefectCollector()
+    )
+    
+    load_info = pipeline.run(source)
+```
+
+### Progress and Summary Reports
+
+If the task is executed the `PrefectCollector` will create progress artifacts that will update
+during a stage as well as summary report after a pipeline step has completed.
+For example during extract you will see how many resources have been extracted and once the extract 
+is completed you will see how many rows have been processed for each resource: 
+
+![Prefect Extract Artifacts](https://storage.googleapis.com/dlt-blog-images/docs-prefect-extract-artifact.png)
+
+Also, each summary report contains basic information about the pipeline, the destination the execution 
+environment and system resources (CPU, memory) usage.
+
+### Schema Change Reports
+
+The `PrefectCollector` will also create artifacts when schema changes are detected.
+This is useful to see which tables and columns have changed between runs.
+
+![Prefect Schema Change Detection](https://storage.googleapis.com/dlt-blog-images/docs-prefect-schema-change-artifact.png)
+
+### Logging
+
+Prefect has built-in functionality to [include logs from other libraries](https://docs.prefect.io/v3/advanced/logging-customization#include-logs-from-other-libraries) and display them as part of their UI.
+
+You can tell prefect to include `dlt`'s logs by setting the corresponding prefect environment variable, for example by adding this to your `.env` file:
+```sh
+PREFECT_LOGGING_EXTRA_LOGGERS=dlt
+```
+
+You should now see dlt's logs in the prefect UI and be able to [query them with the prefect CLI](https://docs.prefect.io/v3/how-to-guides/workflows/add-logging#access-logs-from-the-command-line).
+
+
+## Runner integration
+
+The `PrefectCollector` integrates seamlessly with the [dlt+ runner](../production/pipeline-runner.md). 
+
+If you are running your pipelines with the dlt+ runner, information about the pipelines run configuration and the trace pipeline 
+will also be included in the artifacts in the `Run Configuration` section.
+
+### Pipeline Retries
+
+Prefect retry-mechanism is not a perfect fit for dlt pipelines.
```

Comment:
I suggest first to mention that dlt+ Runner fixes the problem and then explain it :) 

```
The dlt+ Runner provides a retry configuration that ensures pipeline state and intermediate results are preserved across retry attempts.

This is important because Prefect’s default retry mechanism is not optimized for dlt pipelines. During execution, dlt generates intermediate files in the pipeline’s working directory. If a run fails and Prefect retries it, those files may not be available anymore. For example, if the retry happens on a different worker node or inside an ephemeral Docker container.

To do so, ...
```